### PR TITLE
AI Artificial Intelligence Search Methods For Problem Solving

### PR DESCRIPTION
**Pull request by AJain2810**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo
Added resource

### Description
AI Artificial Intelligence Search Methods For Problem Solving

### Why is this valuable (or not)?
In depth knowledge of using search strategies for AI.

### How do we know it's really free?
Distributed freely on YouTube.

### For book lists, is it a book? For course lists, is it a course? etc.
course
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-hi.md [hunk 1]
```diff
@@ -159,9 +159,11 @@
 ### Databases
 
 * [DBMS (Database Management System)Complete Playlist](https://www.youtube.com/playlist?list=PLxCzCOWd7aiFAN6I8CuViBuCdJgiOkT2Y) - Gate Smashers
+* [DBMS Database Managament System- Complete Course in Hindi](https://youtube.com/playlist?list=PLmXKhU9FNesR1rSES7oLdJaNFgmuj0SYV) - Knowledge Gate
 * [DBMS Placements Series 2022](https://www.youtube.com/playlist?list=PLDzeHZWIZsTpukecmA2p5rhHM14bl2dHU) - CodeHelp by Babbar
 
 
+
```
Comment (metabiswadeep) [inline courses/free-courses-hi.md:166]:
```suggestion
```

**Review by metabiswadeep — CHANGES_REQUESTED** [review]

File: courses/free-courses-hi.md (hunk 1 above)
Comment (metabiswadeep) [reply courses/free-courses-hi.md:166]: Dont mess with the spaces

Comment (AJain2810) [reply courses/free-courses-hi.md]: @metabiswadeep : Fixed the spacing issue

**Comment by eshellman** [comment]
please resolve conflicts.

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -193,6 +193,7 @@
 * [AI Fundamentals](https://www.udacity.com/course/ai-fundamentals--ud099) - Microsoft Azure (Udacity)
 * [Aml-2018 Ambient Intelligence](https://www.youtube.com/playlist?list=PLqRTLlwsxDL8fUcY2Y54sITILyJcTySpC) - Fulvio Corno, Luigi De Russis, Alberto Monge Roffarello @ Politecnico di Torino
 * [Artificial Intelligence on Google Cloud Platform](https://www.youtube.com/playlist?list=PL3N9eeOlCrP6Nhv4UFp67IsQ_TVDpXqXK) - Srivatsan Srinivasan @ AIEngineering (YouTube)
+* [Artificial Intelligence Search Methods For Problem Solving](https://youtube.com/playlist?list=PLbMVogVj5nJSFZoiF6RDqyz_m6Srjx_MY) - NPTEL HRD
```
Comment (davorpa) [inline courses/free-courses-en.md:196]:
```suggestion
* [Artificial Intelligence Search Methods For Problem Solving](https://youtube.com/playlist?list=PLbMVogVj5nJSFZoiF6RDqyz_m6Srjx_MY) - nptelhrd
```

File: courses/free-courses-hi.md [hunk 3]
```diff
@@ -168,8 +168,9 @@
 
 ### Databases
 
-* [DBMS (Database Management system) Complete Playlist](https://www.youtube.com/playlist?list=PLxCzCOWd7aiFAN6I8CuViBuCdJgiOkT2Y) - Gate Smashers
-* [DBMS Placements Series 2022](https://www.youtube.com/playlist?list=PLDzeHZWIZsTpukecmA2p5rhHM14bl2dHU) - CodeHelp - by Babbar
+* [DBMS (Database Management System) \| Complete Playlist](https://www.youtube.com/playlist?list=PLxCzCOWd7aiFAN6I8CuViBuCdJgiOkT2Y) - Gate Smashers
+* [DBMS Database Managament System \| Complete Course in Hindi](https://youtube.com/playlist?list=PLmXKhU9FNesR1rSES7oLdJaNFgmuj0SYV) - Knowledge Gate
+* [DBMS Placements Series 2022](https://www.youtube.com/playlist?list=PLDzeHZWIZsTpukecmA2p5rhHM14bl2dHU) - CodeHelp by Babbar
```
Comment (davorpa) [inline courses/free-courses-hi.md:173]: - Don't invent titles/YT channels. Use the ones as provided by original source
- Alphabetize then

```suggestion
* [3.1 DBMS In Hindi (Complete Playlist)](https://youtube.com/playlist?list=PLmXKhU9FNesR1rSES7oLdJaNFgmuj0SYV) - Knowledge Gate
* [DBMS (Database Management System) Complete Playlist](https://www.youtube.com/playlist?list=PLxCzCOWd7aiFAN6I8CuViBuCdJgiOkT2Y) - Gate Smashers
* [DBMS Placements Series 2022](https://www.youtube.com/playlist?list=PLDzeHZWIZsTpukecmA2p5rhHM14bl2dHU) - CodeHelp - by Babbar
```

**Review by davorpa — APPROVED** [review]

**Review by metabiswadeep — APPROVED** [review]
Fine from my side

**Pull request merged**